### PR TITLE
Add instructions for using django-storages with Scaleway Object storage

### PR DESCRIPTION
**Pull request by mrchrisadams**
hi there!

Thank you for creating Django Storage - I figured out how to use it with [Scaleway Object storage](https://www.scaleway.com/en/docs/storage/object/quickstart/) this week for a project, which is broadly compatible with AWS S3 storage, and I wanted to share this back.

~~I'm a little rusty with my restructured text, so I've made this draft PR, and once I figure out how to format the code example in the docs, I'll remove the draft status, ready for review.~~

I think I've updated it now.

### CODE REVIEW

File: docs/backends/scaleway.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Scaleway
+=============
+
+Scaleway Object storage implements the S3 protocol. To use it follow the instructions in the :doc:`Amazon S3 docs <amazon-S3>` with the important caveats that you must:
+
+- Set ``AWS_BUCKET_NAME`` to the Bucket you want write to (such as ``my-chosen-bucket``)
+- Set ``AWS_S3_REGION_NAME`` to your Scaleway region (such as ``nl-ams`` or ``fr-par``)
+- Set ``AWS_S3_ENDPOINT_URL`` to the value of ``https://{AWS_BUCKET_NAME}.s3.${AWS_S3_REGION_NAME}.scw.cloud``
```
Comment (lvjp) [inline docs/backends/scaleway.rst:8]: The bucket name should not be included into the endpoint.

File: docs/backends/scaleway.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+    from django.core.files.storage import default_storage
+    file = default_storage.open("my_chosen_file.txt", "w")
+    file.write("storage contents")
+    file.close()
+
+If would be written to the following address:
+
+https://my-chosen-bucket.s3.nl-ams.scw.cloud/my-chosen-bucket/my_chosen_file.txt
```
Comment (lvjp) [inline docs/backends/scaleway.rst:22]: If you correctly configure endpoint:

```suggestion
https://my-chosen-bucket.s3.nl-ams.scw.cloud/my_chosen_file.txt
```

or 

```suggestion
https://s3.nl-ams.scw.cloud/my-chosen-bucket/my_chosen_file.txt
```

**Comment by mrchrisadams** [comment]
I've made that change now @lvjp - this look ok now?

**Comment by jschneier** [comment]
The docs have been restructured a bit so that the S3 compatible backends are in their own section. Now that that's inplace I'd be happy to take this if you want to rebase to fit it into that structure.

File: docs/backends/scaleway.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+    from django.core.files.storage import default_storage
+    file = default_storage.open("my_chosen_file.txt", "w")
+    file.write("storage contents")
+    file.close()
+
+If would be written to the following address:
+
+https://s3.nl-ams.scw.cloud/my-chosen-bucket/my_chosen_file.txt
```
Comment (lvjp) [inline docs/backends/scaleway.rst:22]:
```suggestion
If would be written to the following addresses :

https://s3.nl-ams.scw.cloud/my-chosen-bucket/my_chosen_file.txt
https://my-chosen-bucket.s3.nl-ams.scw.cloud/my_chosen_file.txt
```

**Comment by jschneier** [comment]
Superseded by #1419

**Pull request closed without merging**